rocAL/rocAL/source/loaders/video/video_read_and_decode.cpp

```diff
@@ -160,13 +159,13 @@ VideoReadAndDecode::load(unsigned char *buff,
         THROW("Zero image dimension is not valid")
     if (!buff)
         THROW("Null pointer passed as output buffer")
-    if (_video_reader->count_items() < _sequence_count)
-        return VideoLoaderModuleStatus::NO_MORE_DATA_TO_READ;
+    if (_video_reader->count_items() < _batch_size)
```

Comment:
Is the sequence count for a particular batch been handled though we just use the batch_size variable just like images ?

Reply:
Yes, now batch size is used instead of the sequence count